test/pipelinerun_test.go

```diff
@@ -25,17 +25,19 @@ import (
 	"testing"
 	"time"
 
-	"github.com/tektoncd/pipeline/pkg/artifacts"
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/util/wait"
 
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
-	tb "github.com/tektoncd/pipeline/test/builder"
+	"github.com/tektoncd/pipeline/pkg/artifacts"
+
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"knative.dev/pkg/apis"
 	knativetest "knative.dev/pkg/test"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
```

Comment:
the package ordering looks a little funky to me here.  looks like atm it goes:

* stdlib
* k8s.io api machinery
* pipeline/artifacts
* k8s.io other stuff
* pipeline other stuff

Maybe an IDE snafu?

Reply:
Indeed. Ran goimports on this file manually.